mlora/model_llama.py

```diff
@@ -369,14 +390,14 @@ def from_pretrained(path: str,
                 layer.self_attn.o_proj, device=device)
             model.layers_[idx].ffn_ = FeedForward(
                 norm=RMSNorm(layer.post_attention_layernorm.weight.to(
-                    device=device).requires_grad_(False), model.norm_eps_),
+                    device=device).detach(), model.norm_eps_),
```

Comment:
do not use detach function, because we need the tensor in calc graph, but does not need grad.